guides/common/modules/proc_managing-and-monitoring-hosts-using-cockpit.adoc

```diff
@@ -22,8 +15,17 @@ For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{conte
 
 You can now access the full range of features available for host monitoring and management, for example, {LoraxCompose}, through {the-Cockpit}.
 
+.Additional resources
+ifndef::satellite,orcharhino[]
+* For more information about installing {the-Cockpit}, see https://cockpit-project.org/running.html[Running Cockpit].
```

Comment:
My issue with the official guide on installing/running cockpit contain a lot of steps which are unnecessary - we just need the packages to be there.

Reply:
I reworded the reference to be as generic as possible and avoid suggesting that users need to do any extra steps to "install" anything. Other than that, we want to help users find more details about Cockpit: using generic wording and keeping the links in the "Additional resources" section should help with that.

Note that I dropped the other set of links from the Prerequisites section, I believe those were the misleading ones.